NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -262,6 +262,12 @@ public ActionQuery(StandaloneContext standaloneContext)
                         Description = "refill ticket by NCG.",
                         DefaultValue = false,
                         Name = "payNcg",
+                    },
+                    new QueryArgument<ListGraphType<RuneSlotInfoInputType>>
```

Comment:
If you don't want to `[null, null, ...]` as a `runeSlotInfos` argument, it should be like the below.

```suggestion
                    new QueryArgument<ListGraphType<NonNullGraphType<RuneSlotInfoInputType>>>
```